dthm4kaiako/events/models.py

```diff
@@ -252,14 +337,225 @@ def location_summary(self):
         else:
             return None
 
+    def save(self, *args, **kwargs):
+        """Save the event."""
+        return super().save(*args, **kwargs)
+
+    @property
+    def capacity_percentage(self):
+        """Return percentage of capacity that the event is at e.g. at 60% capacity."""
+        if self.capacity is not None:
+            registration_counts = self.registration_status_counts
+            return float(registration_counts['approved']) / self.capacity * 100
```

Comment:
I've moved this function into the `manage_event_view ` and passed the value through as part of the context. This way it can be unit tested easily. I've also floored the value too so that something like 299 attendees out of 300 spots shows as 99% capacity.